R/Task_operators.R

```diff
@@ -323,6 +323,8 @@ recodeY = function(y, type, td) {
     return(as.numeric(2L * (y == td$positive) - 1L))
   if (type %in% c("lcens", "rcens", "icens"))
     return(recodeSurvivalTimes(y, from = td$censoring, to = type))
+  if (type == "multilabel.factor")
```

Comment:
Ideally, this option should also be documented (`recode.target` param of `getTaskData`, l. 229`)